forge-gui/res/cardsfolder/upcoming/stenn_paranoid_partisan.txt

```diff
@@ -0,0 +1,12 @@
+Name:Stenn, Paranoid Partisan
+ManaCost:W U
+Types:Legendary Creature Human Wizard
+PT:2/2
+K:ETBReplacement:Other:ChooseCT
+SVar:ChooseCT:DB$ ChooseType | Defined$ You | Type$ Card | ValidTypes$ Artifact,Planeswalker,Enchantment,Instant,Sorcery | SpellDescription$ As CARDNAME enters the battlefield, choose a card type other than creature or land.
+S:Mode$ ReduceCost | ValidCard$ Card.ChosenType | Type$ Spell | Activator$ You | Amount$ 1 | Description$ Spells you cast of the chosen type cost {1} less to cast.
+A:AB$ ChangeZone | Cost$ 1 W U | Origin$ Battlefield | Destination$ Exile | SubAbility$ DelTrig | SpellDescription$ Exile NICKNAME, Return it to the battlefield under its owner's control at the beginning of the next end step.
+SVar:DelTrig:DB$ DelayedTrigger | Mode$ Phase | Phase$ End of Turn | Execute$ TrigReturn | TriggerDescription$ Return CARDNAME to the battlefield.
+SVar:TrigReturn:DB$ ChangeZone | Defined$ Self | Origin$ Exile | Destination$ Battlefield
```

Comment:
@Northmoc @tool4ever my gut is telling me that the delayed trigger probably needs to remember it

can you test what happens when it is removed from exile and exiled again?

Reply:
yea, otherwise the Self will be battlefield object with old timestamp and fail

Reply:
i thought we were touching delayed trigger again so the host of the trigger isn't lki, so this still works what you don't need to extra remember it?

Reply:
Stuff like _Fog Elemental_ won't need it but here the zone change happens before the trigger is created though